catalog/imaging/transformer/catalog-transformer-nitf/src/main/resources/nitf.xml

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<metacard>
+    <name>nitf</name>
+    <type>ddf</type>
+    <attributes>
+        <!--// nitf-specific fields-->
+        <!--addMultivaluedDescriptor(NITF_VERSION, String.class, true, true, false);-->
+        <attribute>
+            <name>version</name>
+            <type>String</type>
+            <indexed>true</indexed>
+            <stored>true</stored>
+            <tokenized>false</tokenized>
+            <multi-valued>false</multi-valued>
+        </attribute>
+        <!--addMultivaluedDescriptor(FILE_DATE_TIME, Date.class, true, true, false);-->
+        <attribute>
+            <name>fileDateTime</name>
+            <type>Date</type>
+            <indexed>true</indexed>
+            <stored>true</stored>
+            <tokenized>false</tokenized>
+            <multi-valued>false</multi-valued>
+        </attribute>
+        <!--addMultivaluedDescriptor(FILE_TITLE, String.class, true, true, false);-->
+        <attribute>
+            <name>fileTitle</name>
+            <type>String</type>
+            <indexed>true</indexed>
+            <stored>true</stored>
+            <tokenized>false</tokenized>
+            <multi-valued>true</multi-valued>
+        </attribute>
+        <!--addMultivaluedDescriptor(FILE_SIZE, Long.class, true, true, false);-->
```

Comment:
I think some of these comments incorrectly say "addMultivaluedDescriptor" when the attribute isn't multi-valued.